tests/integration/api_v3_product_write.t

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/perl -w
+
+use ProductOpener::PerlStandards;
+
+use Test::More;
+use ProductOpener::APITest qw/:all/;
+use ProductOpener::Test qw/:all/;
+use ProductOpener::TestDefaults qw/:all/;
+
+use File::Basename "dirname";
+
+use Storable qw(dclone);
+
+remove_all_users();
+
+remove_all_products();
+
+wait_dynamic_front();
+
+my $ua = new_client();
+
+my %create_user_args = (%default_user_form, (email => 'bob@gmail.com'));
+create_user($ua, \%create_user_args);
+
+my $tests_ref = [
```

Comment:
I think we should add a comment in all tests where we use this pattern, because that might be surprising for a new comer not to see where expected data is !

```suggestion
# Note: expected results are stored in json files, see execute_api_tests
my $tests_ref = [
```